### DISCUSSION TITLE
Installing an older version of SimNIBS - such as v3.2.3

### DISCUSSION THREAD

**Discussion by eburkevt** [simnibs/simnibs#352] [6. Getting started] [answered]
We'd like to continue to use SimNIBS, but we'd rather not upgrade to version 4.1.0 at this time.

Can we download and install an older versions of SimNIBS?  In our case, we'd like to use version 3.2.3, as our current scripts/code are fully compatible with this older version. Thank you!

**Comment by Teddy0012** [comment, marked as answer]
If I understood correctly, you can click on the 'Code' page above, then click on 'Releases' on the right side, find the version you need, and install it.

**Reply by oulap** [reply]
Yep, the releases are here:

https://github.com/simnibs/simnibs/releases

**Comment by eburkevt** [comment]
Thanks for the replies. I didn't see the download links for the older versions at first.
